### PR TITLE
Render file contents using htmx

### PR DESCRIPTION
**Pull request by bloodearnest**
- **Clean assets before updating**
- **Update to latest job-server components**
- **Use htmx to render the file contents**

### CODE REVIEW

**Comment by bloodearnest** [comment]
Fixes #119

File: airlock/templates/file_browser/index.html [hunk 1]
```diff
@@ -189,8 +192,33 @@
...
+      function setTreeSelection(target, event) {
+          // target here is the hx-get link that has been clicked on
+
+          // remove class from currently selected node
+          document.querySelector('#tree .selected')?.classList.remove('selected');
+
+          // set corrent selected
+          target.classList.add('selected');
+          // ensure parent details container is open, which means clicking on a directory will open containers.
+          target.closest("details").open = true;
```
Comment (evansd) [inline airlock/templates/file_browser/index.html:216]: I love how using the native `<details>` element makes this so easy.

File: airlock/templates/file_browser/index.html [hunk 2]
```diff
@@ -192,12 +192,33 @@
...
+
+  <script type="text/javascript">
+      // keep the selected class up to date in the tree on the client side
+      function setTreeSelection(target, event) {
+          // target here is the hx-get link that has been clicked on
+
+          // remove class from currently selected node
+          document.querySelector('#tree .selected')?.classList.remove('selected');
+
+          // set corrent selected
```
Comment (evansd) [inline airlock/templates/file_browser/index.html:213]:
```suggestion
          // set current selected
```

**Review by evansd — APPROVED** [review]
HTMX really is great, isn't it? Such a big functionality improvement  for so little code.

As the most minor of minor nits, I would have preferred to have the tweaks required to the modal separated out from the commit which updated the assets library so it's really obvious which bits need to be reviewed.

**Pull request merged**